conversion/conversion.go

```diff
@@ -417,14 +418,21 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+func NewDatabaseAdminClientWithEndpoint(ctx context.Context) (*database.DatabaseAdminClient, error) {
+	if endpoint := os.Getenv("SPANNER_API_ENDPOINT"); endpoint != "" {
+		return database.NewDatabaseAdminClient(ctx, option.WithEndpoint(endpoint))
+	}
+	return database.NewDatabaseAdminClient(ctx)
+}
```

Comment:
We don't need to have WithEndpoint suffix in the function name since we don't plan to have two separate functions in any case (one "WithEndpoint", other without).